code/modules/clothing/spacesuits/_spacesuits.dm

```diff
@@ -96,7 +96,13 @@
 
 	// If we got here, it means thermals are on, the cell is in and the cell has
 	// just had enough charge subtracted from it to power the thermal regulator
-	user.adjust_bodytemperature((temperature_setting - user.bodytemperature), use_steps=TRUE, capped=FALSE)
+	var/suit_change = 0
+	var/temp_diff = (temperature_setting - user.bodytemperature)
+	if(temp_diff < 0)
+		suit_change = (log((temp_diff * -1) * 0.16 + 1) * BODYTEMP_COLD_DIVISOR) * -1
+	else
+		suit_change = log(temp_diff * 0.16 + 1) * BODYTEMP_HEAT_DIVISOR
+	user.adjust_bodytemperature(suit_change)
```

Comment:
?

Reply:
I really should have moved the get_temp_diff proc to helpers as this is the same calculation that body temp uses on species. I will push that shortly.